site/docs/v1/tech/apis/system.adoc

```diff
@@ -115,10 +116,10 @@ The format string used to format the date and time columns in the export result.
 When this parameter is omitted a default format of `M/d/yyyy hh:mm:ss a z` will be used. See the https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html[DateTimeFormatter patterns] for additional examples.
 
 [field]#lastNBytes# [type]#[Long]# [optional]#Optional# [default]#defaults to `65,536`#::
-The number of bytes to retrieve from the end of each of the system logs.  When this value is `-1`, the entire log will be downloaded.
+The number of bytes to retrieve from the end of each of the system logs. When this value is `-1`, the entire log will be downloaded.
 
 [field]#zoneId# [type]#[String]# [optional]#Optional# [default]#defaults to **[see description]**#::
-The link:/docs/v1/tech/reference/data-types#time-zone[time zone] used to calculate the current time and build the filename.
+The link:/docs/v1/tech/reference/data-types#time-zone/[time zone] used to calculate the current time and build the filename.
```

Comment:
Same comment as above, if the link ends with a fragment, we should not have a trailing slash. You can test it to be sure, but this should break the link because this tells the browser to look for an `id` on the page of `time-zone/` instead of `time-zone`.